code/game/objects/structures/crates_lockers/closets/secure/bar.dm

```diff
@@ -23,3 +23,26 @@
 	new /obj/item/reagent_containers/food/drinks/bottle/small/beer( src )
 	new /obj/item/reagent_containers/food/drinks/bottle/small/beer( src )
 	new /obj/item/reagent_containers/food/drinks/bottle/small/beer( src )
+
+/obj/structure/closet/secure_closet/cabinet/beer
+	name = "beer closet"
+	desc = "A cabinet stacked end-to-end with six packs of beer."
+	req_access = list(ACCESS_BAR)
+	storage_capacity = 45
+
+/obj/structure/closet/secure_closet/cabinet/beer/fill()
+	new /obj/item/storage/box/fancy/yoke/beer( src )
+	new /obj/item/storage/box/fancy/yoke/beer( src )
+	new /obj/item/storage/box/fancy/yoke/ebisu( src )
+	new /obj/item/storage/box/fancy/yoke/ebisu( src )
+	new /obj/item/storage/box/fancy/yoke/shimauma( src )
+	new /obj/item/storage/box/fancy/yoke/shimauma( src )
+	new /obj/item/storage/box/fancy/yoke/moonlabor( src )
+	new /obj/item/storage/box/fancy/yoke/moonlabor( src )
+	new /obj/item/storage/box/fancy/yoke/earthmover( src )
+	new /obj/item/storage/box/fancy/yoke/earthmover( src )
+	new /obj/item/storage/box/fancy/yoke/whistlingforest( src )
+	new /obj/item/storage/box/fancy/yoke/whistlingforest( src )
+
+/obj/structure/closet/secure_closet/cabinet/beer/unlocked //for awaysites
```

Comment:
should be /horizon subtype specifically for horizon

What you did here was the standard for a long time, but I would like us from now on to move towards the opposite - default no access, and subtypes for horizon. This is cause we have so much more offships and away sites that use everything more than horizon.